packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStep.tsx

```diff
@@ -157,13 +179,29 @@ export const FormStep = ({
                     {fields.length === 0 && (
                         <Center
                             onDrop={item => {
-                                // We don't want to drop steps inside of steps
+                                // We don't want to drop steps inside of steps.
                                 if (item.ui === "step") {
                                     return undefined;
                                 }
-                                handleDropField(item, {
-                                    row: 0,
-                                    index: 0
+                                handleDrop({
+                                    target: {
+                                        type: item.ui,
+                                        id: item.id,
+                                        name: item.name
+                                    },
+                                    source: {
+                                        containerId: item?.container?.id,
+                                        containerType: item?.container?.type,
+                                        position: item.pos
+                                    },
+                                    destination: {
+                                        containerId: formStep.id,
+                                        containerType: "step",
+                                        position: {
```

Comment:
I think that we can use `dropTarget` object here instead of `formStep.id` and `step`.
I just need to re-check it.

Reply:
I have checked whether we can `dropTarget` here, and I figured out that we cannot do that because value for `dropTarget` is being taken from `destination` propetry.